### PR TITLE
Feature/ds 232 react tabs

### PR DESCRIPTION
**Pull request by literat**

### CODE REVIEW

File: packages/web-react/src/components/Tabs/README.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+### TabItem
+
+Tab list item
+
+**Available props**
+
+| Name       | Type               | Description           |
+| ---------- | ------------------ | --------------------- |
+| `forTab`   | `string`, `number` | Identification of tab |
+| `href`     | `string`           | External link         |
```
Comment (adamkudrna) [inline packages/web-react/src/components/Tabs/README.md:66]: Shouldn't we note these two options are mutually exclusive? Looking into the implementation, I am even considering making it two separate components…

File: packages/web-react/src/components/Tabs/README.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
+# Tabs
+
+The Tab component is used to organize related content. User can navigate between
+groups of information in tabbable regions.
+
+## Tab
+
+```jsx
+<Tabs defaultSelectedTab={1}>
+  <Tab>
```
Comment (adamkudrna) [inline packages/web-react/src/components/Tabs/README.md:10]: Maybe `<TabList>` would better describe the purpose? I mean seeing `Tab` inside `Tabs` is OK only until I realize that `Tab` is not an item but just a wrapper for items.

**Review by adamkudrna — COMMENTED** [review]
I left just a few notes to think about, feel free to merge without my ✅.

File: packages/web-react/src/components/Tabs/README.md (hunk 2 above)
Comment (literat) [reply packages/web-react/src/components/Tabs/README.md]: Ok, renamed to `TabList`.

**Pull request merged**